app/scripts/controllers/configurations/AddressFormController.js

```diff
@@ -0,0 +1,88 @@
+(function (module) {
+    mifosX.controllers = _.extend(module, {
+        AddressFormController: function ($scope, resourceFactory, routeParams, location) {
+
+            $scope.formData={};
+            $scope.addressTypes=[];
+            $scope.countryOptions=[];
+            $scope.stateOptions=[];
+            $scope.addressTypeId={};
+            entityname="ADDRESS";
+            $scope.editable=false;
+            clientId=routeParams.id;
+            resourceFactory.clientaddressFields.get(function(data){
+                $scope.addressTypes=data.addressTypeIdOptions;
+                $scope.countryOptions=data.countryIdOptions;
+                $scope.stateOptions=data.stateProvinceIdOptions;
+            }
+            )
+
+
+            resourceFactory.addressFieldConfiguration.get({entity:entityname},function(data){
+                $scope.addresstypId=data[0].is_enabled;
```

Comment:
Usually, we should not have hard coded indexes like this. Problem with this implementation is in future somebody adds a new type in between these indexes, that person need to comeback and look into these existing indexes also